go/exorelay/exorelay.go

```diff
@@ -65,27 +65,30 @@ func (e *ExoRelay) GetMessageChannel() chan structs.Message {
 }
 
 // Send sends a message with the given options
-func (e *ExoRelay) Send(options MessageOptions) (string, error) {
+func (e *ExoRelay) Send(options MessageOptions) (string, string, error) {
```

Comment:
I updated send on the javascript version to return the message instead of the just the message id. Thoughts on doing the same here?